forge-game/src/main/java/forge/game/ability/effects/RepeatEachEffect.java

```diff
@@ -139,6 +137,36 @@ else if (sa.hasParam("DefinedCards")) {
             }
         }
 
+        if (sa.hasParam("RepeatTypes")) {
+            final String repTypes = sa.getParam("RepeatTypes");
```

Comment:
Just make a normal `getDefinedCards` call on this param and let it sort "ThisTurnCast" or anything else out

Reply:
hmm, looks like getDefinedCards won't do that nicely:
![image](https://user-images.githubusercontent.com/103371817/199078225-b49f2abc-4450-424f-ae13-29374b6d37c5.png)
I did borrow the syntax from Count, not from a Defined I saw anywhere :(

Reply:
hmn ok, my bad

Reply:
We could use a getDefined as a stopgap before firing that error message, though